brut.apktool/apktool-lib/src/main/java/brut/androlib/res/ResourcesDecoder.java

```diff
@@ -86,12 +88,15 @@ public void decodeManifest(File outDir) throws AndrolibException {
             } else {
                 LOGGER.info("Decoding AndroidManifest.xml with only framework resources...");
             }
-            InputStream inputStream = inApk.getFileInput("AndroidManifest.xml");
-            OutputStream outputStream = out.getFileOutput("AndroidManifest.xml");
+            inputStream = inApk.getFileInput("AndroidManifest.xml");
+            outputStream = out.getFileOutput("AndroidManifest.xml");
             fileDecoder.decodeManifest(inputStream, outputStream);
 
         } catch (DirectoryException ex) {
             throw new AndrolibException(ex);
+        } finally {
+            closeQuietly(inputStream);
+            closeQuietly(outputStream);
```

Comment:
We have `IOUtils.closeQuietly(out);` from Apache commons which would prevent this new private function.

Reply:
Yeah, I changed it to IOUtils.